src/textual/css/scalar.py

```diff
@@ -367,6 +367,24 @@ def resolve(self, size: Size, viewport: Size) -> Offset:
             round(y.resolve(size, viewport)),
         )
 
+    @classmethod
+    def parse(cls, token: str) -> ScalarOffset:
+        """Create a scalar offset from a string.
+
+        Args:
+            token: String to parse (e.g. '1 2' or '1.0 2.0')
+
+        Returns:
+            New offset
+        """
+        x, y = token.split()
+        # fmt: off
```

Comment:
This comment is to prevent the pre-commit from formatting this.

I don't know if this is accepted here or not, but I thought that:
1. It easier to read this way.
2. It matches the way the rest of the class is formatted (see `ScalarOffset.resolve`).

Reply:
No need to disable black here, add a trailing comma...

Reply:
Thanks for the tip, I fixed it.